test/integration/admin/programs/sidebar/clinician-sidebar.js

```diff
@@ -0,0 +1,323 @@
+import _ from 'underscore';
+
+context('clinician sidebar', function() {
+  specify('edit clinician', function() {
+    cy
+      .server()
+      .routeGroupsBootstrap(_.identity, [
+        {
+          id: '1',
+          name: 'Group One',
+        },
+        {
+          id: '2',
+          name: 'Group Two',
+        },
+      ])
+      .visit()
+      .routeClinicians(fx => {
+        _.each(fx.data, clinician => {
+          clinician.relationships.groups = {
+            data: [
+              {
+                type: 'groups',
+                id: '1',
+              },
+              {
+                type: 'groups',
+                id: '2',
+              },
+            ],
+          };
+        });
+
+        fx.data[0].attributes.name = 'Test Clinician';
+        fx.data[0].attributes.email = 'test.clinician@roundingwell.com';
+        fx.data[0].id = '1';
+        fx.data[0].relationships.role.data.id = '11111';
+
+        return fx;
+      })
+      .routeClinician(fx => {
+        fx.data.attributes.name = 'Test Clinician';
+        fx.data.attributes.email = 'test.clinician@roundingwell.com';
+        fx.data.id = '1';
+        fx.data.relationships.role.data.id = '11111';
+        fx.data.relationships.groups = {
+          data: [
+            {
+              type: 'groups',
+              id: '1',
+            },
+            {
+              type: 'groups',
+              id: '2',
+            },
+          ],
+        };
+
+        return fx;
+      })
+      .navigate('/clinicians')
+      .wait('@routeClinicians');
+
+    cy
```

Comment:
In the sidebar tests let's navigate to the sidebar instead of using the list.  In the list test file let's test that when you click the list item or the add button that it routes to the right place, and stop there.